### PR TITLE
[electrophysiology_browser] Revised EEG filters + Store reference fix

### PR DESCRIPTION
**Pull request by jeffersoncasimir**
This PR does the following:

- Uses a revised set of coefficients values for the filters, depending on the recording's sampling frequency
- Uses multiple store references when there are multiple recordings. It would previously get overwritten.

The following script was used to generate the coefficients:

```
samp = 512; % Change to target frequency
order = 3;

low_pass_items = [15 20 30 40 60];
high_pass_items = [0.5 1 5 10];

for i=1:length(low_pass_items)
        disp(strcat(num2str(low_pass_items(i)), ' low pass'));
        [b, a] = butter(order-1, (low_pass_items(i) / samp), 'low') % argument is (order - 1)
end

for i=1:length(high_pass_items)
        disp(strcat(num2str(high_pass_items(i)), ' high pass'));
        [b, a] = butter(order-1, (high_pass_items(i) / samp), 'high') % argument is (order - 1)
end
```

### CODE REVIEW

**Review by laemtl — APPROVED** [review]

**Comment by driusan** [comment]
@jeffersoncasimir I can't merge this because of a conflict, can you rebase?

File: modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx [hunk 1]
```diff
@@ -303,11 +305,15 @@ const EEGMontage = (
...
                 mouseUp={dragEnd}
                 mouseLeave={dragEnd}
+                chunksURL={chunksURL}
               >
                 <Montage3D />
               </ResponsiveViewer>
             :
-              <ResponsiveViewer>
+              <ResponsiveViewer
+                // @ts-ignore
```
Comment (driusan) [inline modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx:314]: Why is this being ignored? If ts flags something it's usually for a reason.

Comment (jeffersoncasimir) [reply modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx:314]: I am not sure why. It was not introduced by this PR (or myself) and there seem to be many instances of `// @ts-ignore` throughout the module

Comment (driusan) [reply modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx:314]: Huh? This line is being introduced by you in this PR.

Comment (jeffersoncasimir) [reply modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx:314]: You're right. However it is also present 11 lines above, as well as in the other `<ResponsieViewer>` component in `SeriesRenderer.tsx`.

It seems to be related to `export default withParentSize(ResponsiveViewer);` at the end of `ResponsiveViewer.tsx`.

Here is the error I get when I remove the `// @ts-ignore` line you pointed out:
Screenshot 2024-02-27 at 3 17 54 PM

Comment (driusan) [reply modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx:314]: That sounds like a real error that needs to be investigated.

Comment (jeffersoncasimir) [reply modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx]: I believe I found an appropriate solution. It has been pushed to this PR.

**Pull request merged**